engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -134,14 +135,12 @@ object WorkflowActor {
 /**
   * Class that orchestrates a single workflow.
   */
-class WorkflowActor(workflowId: WorkflowId,
+class WorkflowActor(val workflowId: WorkflowId,
```

Comment:
With `case class`es the constructor parameters automatically become attributes of the class, and hence override / implement potentially unimplemented variables in super classes/traits.
With normal classes this is not true, adding the val makes the workflowId an attribute of the class.

```scala
scala> class A(yo: String)
defined class A

scala> val a = new A("hey")
a: A = A@51016012

scala> a.yo
<console>:13: error: value yo is not a member of A
       a.yo
         ^

scala> class A(val yo: String)
defined class A

scala> val a = new A("hey")
a: A = A@7d9d1a19

scala> a.yo
res2: String = hey 
```